src/core/REPL.pm

```diff
@@ -57,14 +59,14 @@ do {
         my &readline    = $WHO<&readline>;
         my &add_history = $WHO<&add_history>;
 
-        method readline(Mu \SELF, Mu \super, Mu \stdin, Mu \stdout, Mu \prompt) {
+        method repl-read(Mu \prompt) {
             my $line = readline(prompt);
 
             if $line.defined {
                 add_history($line);
             }
 
-            $line // nqp::null_s()
+            $line;
```

Comment:
This is a total nitpick on my part and by no means should you feel compelled to follow it - I'm just pointing out my style and attempting to justify it.

I prefer that if implicit returns are used (as `$line` is used here), that the semicolon be omitted from the final line of the block.  This may seem silly, but to me, it a) serves as a hint to the reader, communicating that
the intent is to return that value, and b) if someone is working on the code and adds a line after the implicit return (for debugging or whatever), they get a compiler error rather than suddenly having the routine they just modified return a wildly different value.